### PR TITLE
Fix get_discount function return  type

### PR DESCRIPTION
**Pull request by sultann**
`get_discount` function returns int not array.

### All Submissions:

* [ ] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [ ] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Closes # .

### How to test the changes in this Pull Request:

1.
2.
3.

### Other information:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### CODE REVIEW

File: includes/class-wc-discounts.php [hunk 1]
```diff
@@ -159,7 +159,7 @@ public function get_items_to_validate() {
 	 * @since  3.2.0
 	 * @param  string $key name of discount row to return.
 	 * @param  bool   $in_cents Should the totals be returned in cents, or without precision.
-	 * @return array
+	 * @return int
```
Comment (claudiosanches) [inline includes/class-wc-discounts.php]: In fact it returns `float` and not `int`.

**Review by claudiosanches — CHANGES_REQUESTED** [review]
That method returns `float`.

File: includes/class-wc-discounts.php (hunk 1 above)
Comment (sultann) [reply includes/class-wc-discounts.php]: @claudiosanches thank you for pointing that. I change to  float.

**Review by claudiosanches — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**